src/ontology/cl-edit.owl

```diff
@@ -29538,7 +29538,7 @@ SubClassOf(obo:CL_4023075 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P60041))
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Martinotti_cell&oldid=9947732"^^xsd:string) obo:IAO_0000115 obo:CL_4023076 "An interneuron that has Martinotti morphology. These interneurons are scattered throughout various layers of the cerebral cortex, sending their axons up to the cortical layer I where they form axonal arborization.")
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023076 <http://orcid.org/0000-0001-7258-9596>)
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023076 "ILX:0107375")
-AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023076 "nifext_55")
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023076 "NIFSTD:nifext_55")
```

Comment:
Why not `NIFSTD:55`?

Reply:
Because there're others like this

```
AnnotationAssertion(oboInOwl:hasDbXref obo:CL_1001435 "NIFSTD:nifext_122"^^xsd:string)
```

Reply:
According to your ticket #1607 

```
@prefix NIFEXT: <http://uri.neuinfo.org/nif/nifstd/nifext_>
```

So your example expands to http://uri.neuinfo.org/nif/nifstd/nifext_nifext_55

Which is wrong. So maybe then, all NIFSTD curies are wrong!

Reply:
They expand to nothing because they're all strings?

Reply:
But if the prefix is `@prefix NIFSTD: <http://uri.neuinfo.org/nif/nifstd/>` would work.

Reply:
Yes, ignore that minor detail :D

Reply:
If you want an explanation ask again in Tech call

Reply:
Yesh, I agree @anitacaron NIFSTD would work lets wait for @cthoyt to get his opinion.

Reply:
neither of these are in bioregistry, therefore you can assume I don't have an opinion :p 

but as far as NIF's sprawl of namespaces, I think this is because nobody really knows what any of these do. Can you spot check that these references go to things that make sense if you use the NIFSTD or NIFEXT URI format strings to expand to URIs?

Reply:
Would one of you be willing to submit a bioregistry new prefix request then?

Reply:
oh boy ☹️

Reply:
Just to clarify and resolve this comment, we agree to use the prefix NIFEXT that is going to be added in the bioregistry via https://github.com/biopragmatics/bioregistry/issues/404.

Reply:
Lets wait a moment on @tgbugs to reply to that issue.